CONTRIBUTING.md

```diff
@@ -13,6 +13,18 @@
 ### Testing workflow
 Have a look at `test/README.md` to see how testing works. Run `npm run watch:test` while developing. Have fun! :)
 
+### Tips for working with Joi
+- Use `!!` at the beginning of your custom error string to get rid of the key at the beginning of the error message. Example:
```

Comment:
This is actually awesome sauce; why did I not know about this before? :)
(Means that we can get rid of pesky error messages that look  like "`"value" bla bla`", changing them to "`bla bla`").